src/generateMarkdown.js

```diff
@@ -38,18 +38,25 @@ const generateTitle = name => '`' + name + '`' + '\n===\n'
 
 const generateDescription = description => description + '\n'
 
-const generatePropType = type =>
-  type.name +
-  (type.value && type.name !== 'shape'
-    ? Array.isArray(type.value)
-      ? `(${type.value.map(v => v.name || v.value).join('&#124;')})`
-      : type.value
-    : '')
+const generatePropType = type => {
+  if (Array.isArray(type.name)) {
+    return `${type.name.map(v => v.name || v.value).join(' &#124; ')}`
```

Comment:
I know this comment wasn't there prior to your change, but now that I'm looking at it with fresh eyes the `&#124;` isn't obvious to the reader. Could you commit the following suggestion just so it's there for the next time?

```suggestion
    return `${type.name.map(v => v.name || v.value).join(' &#124; ')}` //  &#124; = |
```

Reply:
Yes that took me some time to figure out as well so makes sense to leave a comment.